src/utils.js

```diff
@@ -14,7 +14,7 @@ export const getUnit = measurement => {
     return "px";
   }
 
-  return measurement.match(/(?<=[0-9])[^0-9]+$/)[0];
+  return measurement.match(/^([+-]?(?:\d+|\d*\.\d+))([a-z]*|%)$/)[2];
```

Comment:
Can you make the unit part case insensitive?
```suggestion
  return measurement.match(/^([+-]?(?:\d+|\d*\.\d+))([A-Za-z]*|%)$/)[2];
```